client/src/styles/chat-override-theme.tsx

```diff
@@ -7,6 +7,6 @@ The full terms of this copyright and license should always be found in the root
 import React from "react";
 import "styles/chat.overrides.css";
 
-const Theme = () => <React.Fragment></React.Fragment>;
+const Theme = (): JSX.Element => <React.Fragment></React.Fragment>;
```

Comment:
I think this file can be removed as its not being used.

Reply:
4543b43